SDL/Event.cuh

```diff
@@ -10,6 +10,7 @@
 #include "MiniDoublet.cuh"
 #include "PixelTriplet.cuh"
 #include "TrackCandidate.cuh"
+#include "Constants.cuh"
```

Comment:
I guess this is just preference, but Event.cu uses alpaka keywords obviously, so I figured it would be good to explicitly include the Constants.cuh file where the alpaka include is called. I think this could be removed without any issues, but then we would be relying on some of these other files that are included also including Constants.cuh rather than just explicitly including it where it's needed.

Reply:
Ok, my concern was that the re-inclusion of the same file through different `include`s could cause a problem in the long run, as we have seen it was the case for some other files we have been including everywhere, even though it was not needed. But, if you think it is only a matter of preference in this case and it won' cause problems, I am fine with including it.

Reply:
Yea but this case is different though because it's just a constants.cuh file that itself doesn't include any other files in the SDL directory, and every file that includes it actually uses it. The other files that were included everywhere (kerles.cuh for example) had a bunch of cyclical includes and it was never even used by the files that included it lol.